builder-support/specs/pdns-recursor.spec

```diff
@@ -81,15 +81,24 @@ make %{?_smp_mflags} check || (cat test-suite.log && false)
 %install
 make install DESTDIR=%{buildroot}
 
-%{__mv} %{buildroot}%{_sysconfdir}/%{name}/recursor.conf{-dist,}
+%{__cp} %{buildroot}%{_sysconfdir}/%{name}/recursor.{yml-dist,conf}
 %{__mkdir} %{buildroot}%{_sysconfdir}/%{name}/recursor.d
 
 # change user and group to pdns-recursor and add default include-dir
-sed -i \
-    -e 's/# setuid=/setuid=pdns-recursor/' \
-    -e 's/# setgid=/setgid=pdns-recursor/' \
-    -e 's!# include-dir=.*!&\ninclude-dir=%{_sysconfdir}/%{name}/recursor.d!' \
-    %{buildroot}%{_sysconfdir}/%{name}/recursor.conf
+cat << EOF > %{buildroot}%{_sysconfdir}/%{name}/recursor.conf
```

Comment:
I'm confused, aren't we overwriting the file we copied on line 84?

Reply:
Yes, line 84 can be left out. That's a remainder of an earlier version, which installed a full config.